src/targets/maven.ts

```diff
@@ -21,6 +21,9 @@ const GRADLE_PROPERTIES_FILENAME = 'gradle.properties';
  * https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_environment_variables
  */
 const DEFAULT_GRADLE_USER_HOME = join(homedir(), '.gradle');
+export const POM_DEFAULT_FILENAME = 'pom-default.xml';
+const POM_FILE_EXT = '.xml'; // Must include the leading `.`
+const BOM_FILE_KEY_REGEXP = stringToRegexp('/<packaging>pom</packaging>/');
```

Comment:
Why use `stringToRegexp` here where you can simply write a regex???

```js
/<packaging>pom<\/packaging>/
```

FYI, it is also unsafe to try parsing an XML file with a RegEx.